src/backy/tests/test_archive.py

```diff
@@ -60,25 +61,68 @@ def test_load_revisions(backup_with_revisions):
     assert a.history[2].get_parent().uuid == "123-1"
     assert a.history[0].get_parent() is None
     assert a.find_revisions("all") == a.history
-    assert a.find_revisions(1) == [a.find(1)]
+    assert a.find_revisions("1") == [a.find("1")]
+    assert a.find_revisions("trust:verified") == [a.find("123-0")]
+    assert a.find_revisions("2..1") == [a.find("2"), a.find("1")]
+    assert a.find_revisions("123-0..123-1") == [
+        a.find("123-0"),
+        a.find("123-1"),
+    ]
+    assert a.find_revisions("last(tag:daily)..123-1") == []
```

Comment:
this should also allow specifying it "wrong". so i guess the definition for the ellipsis is that it automatically chooses the boundaries properly based on "oldest" goes in this place and "newest" goes in this place